### PR TITLE
feat: add prompt option

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

feat

**Did you add tests for your changes?**

yet to

**If relevant, did you update the documentation?**
yet to

**Summary**
Adds a prompt option to allow users to prevent auto installation

**Does this PR introduce a breaking change?**
no

**Other information**
wip on some fixes and tests

### CODE REVIEW

File: src/installer.js [hunk 1]
```diff
@@ -18,6 +19,8 @@ const defaultOptions = {
   dev: false,
   peerDependencies: true,
   quiet: false,
+  // flip to true
+  prompt: false,
```
Comment (alexander-akait) [inline src/installer.js:23]: Let's do it `true` by default

File: src/installer.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
 const resolve = require('resolve');
-
 const spawn = require('cross-spawn');
-
 const JSON5 = require('json5');
 
+const { green, yellow } = require('colorette');
+
+const prompt = require('./utils/prompt');
```
Comment (alexander-akait) [inline src/installer.js:12]: We can use `method` :smile:

Comment (anshumanv) [reply src/installer.js:12]: can you clarify?

Comment (snitin315) [reply src/installer.js:12]: I think he means, using it as `this.prompt` instead of creating separate util and importing.

Comment (alexander-akait) [reply src/installer.js:12]: Yep :+1:

File: src/installer.js (hunk 1 above)
Comment (anshumanv) [reply src/installer.js]: done

File: src/installer.js (hunk 2 above)
Comment (anshumanv) [reply src/installer.js]: we don't have access to plugin class in the installer so I've inlined the util in installer instead

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
@snitin315 @anshumanv next job https://github.com/webpack-contrib/install-webpack-plugin/issues/94 - we should avoid using `console.log/warn/error` in favor built-in loader everywhere https://webpack.js.org/api/logging/#examples-of-how-to-get-and-use-webpack-logger-in-loaders-and-plugins

**Comment by snitin315** [comment]
On it 👍

**Pull request merged**